apps/cowswap-frontend/src/modules/limitOrders/services/tradeFlow/index.ts

```diff
@@ -85,6 +87,8 @@ export async function tradeFlow(
       signer: provider.getSigner(),
       validTo,
     })
+    cowEventEmitter.emit(CowEvents.ON_POSTED_ORDER, { orderUid: orderId, chainId })
+
```

Comment:
For SC contract wallets, this will trigger every time the place button is clicked, but not necessarily after the wallet receives/returns the tx hash.

Here, the even should be emitted after the order is un-hidden on line 123
![image](https://github.com/cowprotocol/cowswap/assets/43217/adbb980b-7dcd-4142-af06-79d7f14ffc1c)

Reply:
Nice catch! Addressed 1cb058ec9a05283aa7542d8a82845a974454615b